src/services/repo/routes.ts

```diff
@@ -0,0 +1,54 @@
+import { Request, Response } from "express";
+const axios = require('axios')
+
+
+const crypto = require('crypto');
+const fs = require('fs');
+const path = require('path');
+
+var scripts:Map<string, String> = new Map();
```

Comment:
Missing comments and better variable names (ppath?) and I would move this to a function

Reply:
moved it to a function but not sure about variable names